packages/headless/src/features/index.ts

```diff
@@ -244,6 +263,11 @@ export namespace ProductRecommendationsActions {
   export type StateNeededByGetProductRecommendations = StateNeededByGetProductRecommendationsAlias;
 }
 
+import {deselectAllFacets as deselectAllFacetsAlias} from './facets/generic/facet-actions';
+export namespace BreadcrumbActions {
```

Comment:
I exposed the action under `BreadcrumbActions` instead of `GenericFacetActions`. I felt it was a more intuitive name, but I am open to feedback,